lib/src/commands/radio.dart

```diff
@@ -75,6 +76,8 @@ ChatGroup radio = ChatGroup(
             channelId: context.channel.id,
           ).startPlaying();
 
+        SongRecognitionService.instance.currentRadio = radio;
```

Comment:
This breaks down if we're serving two guilds simultaneously; if Guild1 plays `America's Greatest 80s Hits`, that will be stored as the `currentRadio`. If Guild2 plays `Aspen 102.3 FM` after Guild1, the `currentRadio` will be overridden. If we provide the `client` found in the program's entry file to the `SongRecognitionService`, we should be able to get the current radio based on the `guildId`.

Reply:
I think it got solved with the new `setCurrentRadio` and `currentRadio` functions in `SongRecognitionService`!